lib/tbot/workloadidentity/workloadattest/podman/client.go

```diff
@@ -0,0 +1,164 @@
+/*
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// Package podman implements bindings for a small subset of Podman's REST API.
+//
+// We use this instead of importing github.com/containers/podman/v5/pkg/bindings
+// because we only use a couple of endpoints, so it's not worth taking on its
+// transitive dependencies.
```

Comment:
Let me know what you think of this decision too. Given how little of the API we actually need to use, it really didn't seem worth adding a dependency on podman (especially as `pkg/bindings` isn't even isolated into a separate module).

Reply:
Ah yes - I ran into this in the past as well (https://github.com/containers/podman/issues/12548). I think for as long as we only need a handful of the RPCs, maintaining a hand rolled client likely makes the most sense for now.